audio/internal/convert/resampling.go

```diff
@@ -17,14 +17,19 @@ package convert
 import (
 	"io"
 	"math"
+	"sync"
 )
 
-var cosTable = [65536]float64{}
+var lazyInit sync.Once = sync.Once{}
+var cosTable []float64
 
 func init() {
-	for i := range cosTable {
-		cosTable[i] = math.Cos(float64(i) * math.Pi / 2 / float64(len(cosTable)))
-	}
+	lazyInit.Do(func() {
```

Comment:
I don't think this is a lazy initialization.

Reply:
Thanks for pointing this out, Actually I took a reference from this resource [Link](https://cloud.google.com/functions/docs/bestpractices/tips#functions-tips-lazy-globals-go)
I'd be grateful if you can give suggestions on how should I go ahead to implement lazy initialization.

Reply:
So you used a code even you didn't fully understand? That's not good.

And I don't think you understand what #2286 is yet. Please come back when you understand this.